src/main/java/com/cloudbees/plugins/credentials/CredentialsStoreAction.java

```diff
@@ -298,7 +298,7 @@ public ContextMenu getContextMenu(String prefix) {
         ContextMenu menu = new ContextMenu();
         if (getStore().isDomainsModifiable() && getStore().hasPermission(MANAGE_DOMAINS)) {
             menu.add(ContextMenuIconUtils.buildUrl(prefix, "newDomain"),
-                    getMenuItemIconUrlByClassSpec("icon-credentials-new-domain icon-md"),
```

Comment:
it's this line here that breaks the context menu loading for add domain.

I think it's using the icon to determine the url =/.

Needs refactoring